desafio11.sql

```diff
@@ -0,0 +1,2 @@
+SELECT notes FROM northwind.purchase_orders
+WHERE notes LIKE 'Purchase generated based on Order #3%';
```

Comment:
Aqui seria legal colocar um underline, pra indicar que ali teria mais algum caracter, ao invés do %, que pega qualquer um que tenha mais coisa depois... 
Se fosse a Order nº 300 por exemplo, ele pegaria com o " % ", mas não com o " _ ".
```suggestion
WHERE notes LIKE 'Purchase generated based on Order #3_';
```

Reply:
Verdade, nem lembrava da existência desse _ haha só cacei o primeiro método de ignorar caracteres. Mas valeu a ideia, de fato é o ideal